s4e-backend/src/main/java/pl/cyfronet/s4e/geoserver/op/GeoServerOperations.java

```diff
@@ -156,11 +197,36 @@ public void deleteCoverageStore(String workspace, String coverageStore, boolean
     }
 
     public void createS3Coverage(String workspace, String coverageStore, String coverage) {
-        URI url = UriComponentsBuilder.fromHttpUrl(
-                geoServerProperties.getBaseUrl() + "/workspaces/{workspace}/coveragestores/{coveragestore}/coverages")
-                .buildAndExpand(workspace, coverageStore).toUri();
-        val entity = httpEntity(new CreateS3CoverageRequest(workspace, coverageStore, coverage));
-        restTemplate().postForObject(url, entity, String.class);
+        String body = "{\n" +
+                "\"coverage\":{\n" +
+                "   \"name\": " + coverage + ",\n" +
+                "   \"title\":" + coverage + ",\n" +
+                "   \"enabled\":true,\n" +
+                "   \"type\": \"raster\",\n" +
+                "   \"metadata\":\n{\n" +
+                "       \"entry\":[{\n" +
+                "           \"@key\":\"time\",\n" +
+                "           \"dimensionInfo\":{\n" +
+                "               \"enabled\":true,\n" +
+                "               \"presentation\":\"LIST\",\n" +
```

Comment:
Better to use the presentation type with duration, or sth like that, so that not all granules are listed.